<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgFrameSet</class>
 <widget class="QDialog" name="CDlgFrameSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>293</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>366</width>
    <height>293</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>366</width>
    <height>293</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="eName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>31</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>27</y>
     <width>52</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="CWFrameSet" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="btnDeleteFrame">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>20</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Delete Image</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="lgck-builder.qrc">
     <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="btnAddFrames">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>25</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Add New Images</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="lgck-builder.qrc">
     <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CWFrameSet</class>
   <extends>QTreeWidget</extends>
   <header>WFrameSet.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="lgck-builder.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDlgFrameSet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDlgFrameSet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
